sound/soc/sof/ops.h

```diff
@@ -470,6 +470,14 @@ snd_sof_set_mach_params(const struct snd_soc_acpi_mach *mach,
 		sof_ops(sdev)->set_mach_params(mach, dev);
 }
 
+/* client register */
+static inline void
+snd_sof_client_register(struct snd_sof_dev *sdev)
+{
+	if (sof_ops(sdev) && sof_ops(sdev)->client_register)
+		sof_ops(sdev)->client_register(sdev);
+}
+
```

Comment:
With fresh eyes to the patchset, I was initially confused by what client register means here. I.e. is this a DSP-specific way to add a client, or a method to add clients specific to this DSP. No client object is passed in argument, so yeah, the latter is what the implementation does and makes sense in the wider design, but I wonder if the method name could be more descriptive still. Perhaps "register_clients()" -- i.e. to hint that there can be many clients and that it's upto the implementation to know what those clients are (if any).

Reply:
ack!